catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -216,4 +227,32 @@ public void setEnforceWarnings(boolean enforceWarnings) {
     public boolean getEnforceWarnings() {
         return enforceWarnings;
     }
+
+    protected class ValidationResult {
```

Comment:
i would just make errors and warnings publicly accessible and just set them directly. I could be super wrong there, I don't know best practices for tuple implementations